spec/features/ctc/dependents_spec.rb

```diff
@@ -42,7 +42,7 @@ def fill_in_dependent_info(dependent_birth_year)
       select "Social Security Number (SSN)"
       fill_in I18n.t('views.ctc.questions.dependents.tin.ssn_or_atin', name: "Jessie"), with: "222-33-4445"
       fill_in I18n.t('views.ctc.questions.dependents.tin.ssn_or_atin_confirmation', name: "Jessie"), with: "222-33-4445"
-      click_on I18n.t('general.continue')
+      click_on I18n.t('views.ctc.questions.dependents.tin.save_person')
```

Comment:
Thanks @embarnard ! I saw this and didn't submit a fix as quickly as you!

Reply:
I just merged this one commit onto main, so consider rebasing this pull request.